.travis.yml

```diff
@@ -13,7 +13,7 @@ services:
 
 install:
   # Install test dependencies.
-  - pip install ansible==2.7.10 molecule[docker]
+  - pip install ansible==2.9.2 molecule[docker]
```

Comment:
You can switch to "pip install ansible molecule[docker]". The version was only needed due to some issues in 2.8.x, which were fixed in 2.9.1